.gitignore

```diff
@@ -14,5 +14,7 @@ npm-debug.log*
 /backstop_data/ci_report/
 /backstop_data/backstop_scenarios.json
 /backstop_data/packageDependentMap.json
+backstop_data/engine_scripts/
```

Comment:
were there changes to backstop.js that now create locally these files? 
I thought we may want to test using the public CSS website via Jenkins:

Run VRT in CSS against online docs via Jenkins:
https://paper.dropbox.com/doc/Spectrum-CSS-Visual-Regression-Testing--Bl90pyxEw9lIIgs4AnK2RtNVAg-zZtJFcd5DuSOCiGjcdkuB

Reply:
I think these files were generated when I was troubleshooting backstop on my ARM64 machine. I'm not able to access the URLs in the Jenkins instructions, so unfortunately I cannot use that method at present. I can remove these files from my branch and rebase to main if they do not have a purpose.

Reply:
Yeah, let's remove the changes to this file for now, please.

Reply:
New[ commit](https://github.com/adobe/spectrum-css/commit/b243c92da8fd04d3be21d5bf786b333b7fe6d50a) resolves this.